src/App.js

```diff
@@ -1,27 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
 
+// eslint-disable-next-line no-unused-vars
+import Calculator from './components/Calculator';
+
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
```

Comment:
- [ ] Please use a class-based component here. Please go back and have a look at the project requirements.